elements/pl-hidden-hints/pl-hidden-hints.py

```diff
@@ -0,0 +1,102 @@
+import chevron
+import lxml.html
+import prairielearn as pl
+from typing import TypedDict, Tuple, List
+
+
+class HintsDict(TypedDict):
+    """A class with type signatures for the partial scores dict"""
+    html_hints: List[Tuple[int, str]]
+    submission_count: int
+
+
+def prepare(element_html: str, data: pl.QuestionData) -> None:
+    # Set submit count
+    if ('hidden_hints' in data['params']):
+        raise ValueError("Duplicate params key: 'hidden_hints'.")
```

Comment:
I believe this means it's invalid to have multiple `<pl-hidden-hints>` on the same question?

Reply:
This does. It's probably worth removing, so then we just have to ask the client question for an dict key prefix.